sonarqube/resource_sonarqube_project.go

```diff
@@ -60,7 +60,7 @@ func resourceSonarqubeProject() *schema.Resource {
 			"project": {
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
I think we can just remove ForceNew completely, since the default is false

Reply:
Removed!